modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -84,7 +84,7 @@ class NDB_Form_Statistics extends NDB_Form
     ) {
 
         $tpl_data = array();
-        $tpl_data['test_name']  = $_REQUEST['test_name'];
+        $tpl_data['test_name']  = htmlspecialchars($_REQUEST['test_name']);
```

Comment:
@driusan Would it make sense to create a temporary Utility function to get `$_REQUEST` variables that would take the key as a parameter and return the `htmlspecialchar` string in return. Just until we find a better solution?